examples/counter/web/src/lib/rooch_types.yml

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
We can consider auto-generate rooch_types.yml in the ci.

Reply:
I have tried to use serde-generate to generate TS, but an UnknownFormatInContainer error will appear. Through exclusion, it is determined that the indirect recursive definition of pub type_params: TypeTag in the StructTag structure causes serde_generate to report an error.

Reply:
If we use serde-generate to generate rooch_types.yaml should report the same error. Later, we need to fix the bug of serde-generate before we can generate rooch_types.yaml through CI.

Reply:
help to record an issue.

Reply:
> help to record an issue.

OK